vissl/models/__init__.py

```diff
@@ -3,11 +3,23 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from vissl.models.base_ssl_model import BaseSSLMultiInputOutputModel
-from vissl.models.model_helpers import (  # noqa
-    convert_sync_bn,
-    is_feature_extractor_model,
-)
+"""
+Here we create all the custom models required for SSL.
+"""
+
+from pathlib import Path
+
+from classy_vision.generic.registry_utils import import_all_modules
+from classy_vision.models import MODEL_REGISTRY
```

Comment:
If possible, can we not use the classyvision's MODEL_REGISTRY? 

I would recommend following the pattern in https://github.com/facebookresearch/vissl/blob/master/vissl/trainer/train_steps/__init__.py#L22 for the #build_model method. https://github.com/facebookresearch/vissl/blob/master/vissl/models/__init__.py#L13.